### DISCUSSION TITLE
More results when scrolling?

### DISCUSSION THREAD

**Discussion by nipjack** [photoprism/photoprism#1491] [Q&A] [answered]
Hello,

thanks for the good work, I like the app! One thing bothers me though: I cannot quickly scroll through my photos, because there are only 50 images loaded at once. Is there a possibility to increase the limit/smooth scrolling?

Thanks a lot!

**Comment by lastzero** [comment, marked as answer]
It requires a different approach, especially to keep memory usage of your browser at a reasonable size.

We're going to develop a timeline view for this, see https://github.com/photoprism/photoprism/issues/152

**Reply by nipjack** [reply]
Thanks for the fast reply!
